ginga/misc/plugins/Histogram.py

```diff
@@ -441,10 +444,26 @@ def cut_levels(self):
         try:
             loval = float(self.w.cut_low.get_text())
             hival = float(self.w.cut_high.get_text())
+        except Exception as e:
+            errmsg = 'Error cutting levels: {0}'.format(str(e))
+            self.fv.showStatus(errmsg)
+            self.logger.error(errmsg)
+            return
 
+        # This is needed to avoid UserWarning from matplotlib
+        if loval == hival:
```

Comment:
I wonder if it would be better to compare these *before* they are converted to `float`s. Because of the complications of comparing two floating point values exactly.  There are a few other places this sort of thing needs to be fixed in ginga, but I'm just thinking this is a chance to avoid adding another such case.